extensions/workflows/sections/clause_9_deployable_workflow.adoc

```diff
@@ -0,0 +1,37 @@
+== Deployable Workflows Conformance Class
+
+=== Requirement Class _Deployable Workflows_
+
+The _Deployable Workflows_ conformance class specifies how a workflow execution request as defined in
+_OGC API - Processes - Part 1: Core_, with or without the capabilities defined in other conformance classes
+of this _Part 3_ extension, can be used as an application package payload to deploy new processes using
+_OGC API - Processes - Part 2: Deploy, Replace, Undeploy_.
+
+It also specifies how to parameterize inputs to the workflow as inputs to the deployed process.
+
+include::../requirements/requirements_class_deployable-workflows.adoc[]
+
+The following requirements describe the ability of an implementation to deploy a new process
+defined as an _OGC API - Processes - Part 1: Core_ execution request, parameterized as external input
+to the process using `{ "$input" : "{input name}" }` objects,
+with _OGC API - Processes - Part 2: Deploy, Replace, Undeploy_.
+
+A minimal but adequate process description for the resulting process can automatically be generated
+based on the context where the `"$input"` objects appear within the workflow definition, allowing
+to describe all inputs to the process and its outputs by inferrence from the source process descriptions
+and/or input data.
+
+The workflow definition could either directly be the payload of the `POST` request to `/processes`,
+or be part of the `executionUnit` of an OGC Application Package as defined in Part 2's _OGC Application Package_
+conformance class.
+
+NOTE: Different media types than `application/json` or some sort of https://github.com/opengeospatial/ogcapi-common/issues/8[negotiation by profile] will be required
+to distinguish between an _OGC API - Processes_ execution request and an OGC Application Package JSON.
```

Comment:
In fact, the CWL media type does not apply here, as the possibilities for Deployable Workflows conformance classes in Part 3 right now are all either an OGC API - Process execution request JSON, or the OGC Application Package JSON.

The CWL media type would apply to the Part 2 conformance class for CWL.

Reply:
For OGC AppPkg, it would be this type: https://github.com/opengeospatial/ogcapi-processes/blob/master/extensions/deploy_replace_undeploy/standard/sections/clause_9_media_types.adoc

I'm having a hard time seeing how a request using that definition could be used for the Execute endpoint though. The inputs/outputs items used for the execution would have to be injected somewhere in there. That would make this schema even more convoluted than it already is.

Reply:
@fmigneault  Thanks I will update to use `application/ogcapppkg+json`, however it is [not yet registered with IANA](https://www.iana.org/assignments/media-types/media-types.xhtml). Is someone currently tasked with this?

> I'm having a hard time seeing how a request using that definition could be used for the Execute endpoint though.

The _Deployable Workflows_ conformance class is about submitting an execution request to deploy a workflow, so this is POSTed to `/processes` (part 2). But if you mean how you then execute that deployed process from an excution end-point, the inputs are indeed injected in there with `"$input"` ( https://github.com/jerstlouis/ogcapi-processes/blob/part3-update/extensions/workflows/requirements/deployable-workflows/REQ_deployable-workflows_parameterized-inputs.adoc ).

About the outputs, you just reminded me (thank you) of [thoughts from our previous conversation](https://github.com/opengeospatial/ogcapi-processes/issues/279#issuecomment-1048109824) wher I had realized we also need an `"$output"` to support returning multiple outputs from separate processes and rename outputs from the source process generating htem. The default if not specified would be to return all outputs from the top-level process.

Reply:
I'm not aware of anyone tasked to do it.